config/application.rb

```diff
@@ -65,7 +65,7 @@ class Application < Rails::Application
     config.action_view.automatically_disable_submit_tag = false
 
     # Configure sensitive parameters which will be filtered from the log file.
-    config.filter_parameters += [:content, :password, :terms_of_service_non_production]
```

Comment:
Should we keep `:password` still?

Reply:
from the other file:
> partially matched (e.g. passw matches password) 

so no, that would be redundant